resources/views/processes/index.blade.php

```diff
@@ -16,9 +16,10 @@
                                placeholder="{{__('Search')}}...">
                     </div>
                     <div class="col-md-4 d-flex justify-content-end align-items-center col-sm-12 actions">
-                        <a href="#" class="btn btn-action"><i class="fas fa-plus"></i> {{__('Process')}}</a>
+                        <a href="#" @click="openModalProcess" class="btn btn-action"><i class="fas fa-plus"></i> {{__('Process')}}</a>
```

Comment:
Use the built in open method.

Reply:
yes, but i need to call function because a rest call to rest api